src/test/logic/addressBook/utils/addressBookUtils.test.ts

```diff
@@ -0,0 +1,89 @@
+import { Map, List } from 'immutable'
```

Comment:
This test is placed in src/test while other ones are placed  close to where the actual functions are defined. Can we follow that approach for this one too?
Also, there are some other tests that are placed in src/test, could you please check it?

Reply:
done